addon-resizer/go.mod

```diff
@@ -1,22 +1,23 @@
 module k8s.io/autoscaler/addon-resizer
 
 go 1.21
+toolchain go1.22.2
```

Comment:
I don't think this change is correct. It depends on what we build with, but right now this is being set to the go version dependabot is using.

Reply:
yeah, you are right @kwiesmueller, both should be same.

The toolchain is added now, previously it was not there.
https://github.com/kubernetes/autoscaler/blob/addon-resizer-release-1.8/addon-resizer/go.mod

It may work if we change go v1.21 to v1.21.0
https://go.dev/doc/toolchain

Reply:
As per above comment, removing `lgtm` from the PR

/lgtm cancel

Reply:
Hi, @kwiesmueller your thoughts on this.